database/migrations/2017_11_20_142551_create_item_menu_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateItemMenuTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('item_menu', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('item_id')->index();
```

Comment:
Why are you indexing the foreign keys?

Reply:
I have removed the index on the feature/add-meal branch. I realized the pivot table was named wrongly. It was item_menu instead of item_meal.